docs/guides/src/main/server/fips.adoc

```diff
@@ -167,37 +167,36 @@ in the RHEL documentation for the details.
 The keycloak container image will be just automatically in fips mode when executed from the host in FIPS mode. 
 However, make sure that the Keycloak container also uses BCFIPS jars (instead of BC jars) and proper options when started.
 
-Regarding this, it is best to build your own container image as described in the <@links.server id="containers"/> and tweak it to use BCFIPS etc.
+Regarding this, it is best to build your own container image and tweak it to use BCFIPS etc.
 
-For example in the current directory, you can create sub-directory `files` and add:
+For example in the current directory, you can create sub-directory `keycloak` and add unpacked keycloak server, which you already prepared based on the steps above. So assumption is that it contains:
 
-* BC FIPS jar files as described above
-* Custom keystore file - named for example `keycloak-fips.keystore.bcfks`
-* Security file `kc.java.security` with added provider for SAML
+* BC FIPS jar files as described above in directory `providers`
+* Custom keystore file in `conf/server.keystore`
+* Security file `kc.java.security` with added provider for SAML in the `conf` directory
 
 Then create `Dockerfile` in the current directory similar to this:
 
 .Dockerfile:
 [source, dockerfile]
 ----
-FROM quay.io/keycloak/keycloak:latest as builder
-
-ADD files /tmp/files/
+FROM registry.access.redhat.com/ubi9/ubi:latest as builder
```

Comment:
It should be possible to use `quay.io/keycloak/keycloak:latest` as a base image and copy from it.

```
FROM quay.io/keycloak/keycloak:latest AS keycloak-base

FROM fedora:latest
ENV LANG en_US.UTF-8

# copy dist from base
COPY --from=keycloak-base --chown=1000:0 /opt/keycloak /opt/keycloak

RUN echo "keycloak:x:0:root" >> /etc/group && \
    echo "keycloak:x:1000:0:keycloak user:/opt/keycloak:/sbin/nologin" >> /etc/passwd

RUN dnf update -y && \
   dnf install -y --nodocs java-17-openjdk-headless

USER 1000

EXPOSE 8080
EXPOSE 8443

ENTRYPOINT [ "/opt/keycloak/bin/kc.sh" ]
```

It would also be nice if the example could download the artifacts directly from Maven Central and add them to the providers dir?

I'm not sure but, in your example, is the `keycloak` user running the process?

But if you do not want to invest time on this example, feel free to ignore the comments.